README.md

```diff
@@ -17,94 +25,136 @@
 
 ### Chapter 2: Getting Started with Dash
 
- - Structure of a Dash app: data, layout, callback
- - Installation
+ - Structure of a Dash app
+ - Installation and setup
+ - Create your first Dash app
+ - Live updates and debugging
 
-### Chapter 3: Dash Components and Layout
+### Chapter 3: Dash Components and Layouts
```

Comment:
Somewhere in here we should have html components as well, even if it's just the html.div and html.label. I think this is needed before, or while, we explain how to design the app layout.

Reply:
I wondered about this - I had a look through the HTML components, and I couldn't imagine using any one of the HTML components at this stage.

Let's leave this as an open issue for further exploration and discussion ?

Reply:
yes, ok. I am all in favor of not teaching html; and just using dbc to show them how to define layout.

Reply:
perfect!